index.js

```diff
@@ -53,9 +60,6 @@ Template.prototype.init = function(dest, callback) {
     var vars = self.mod;
     var keys = Object.keys(vars);
 
-    if (dest) {
```

Comment:
i was initially checking if self.values was empty but then switched to just having a flag fromJson. putting back.